doc/source/applications.rst

```diff
@@ -2,35 +2,58 @@
+Lethe has several applications that can be used depending on the problem to be solved. Once Lethe is compiled, all the executables to these applications become available. In the following table a brief description of each application is provided:
 
 .. list-table::
     :header-rows: 1
-    :widths: 40 60 
+    :widths: 40 40 60 
 
-    * - Application
+    * - Physics 
+      - Application
       - Description
-    * - ``lethe-fluid``
-      - This application solves the Navier-Stokes equations in a monolithic way. **This is the most robust fluid dynamics solver in Lethe**.
-    * - ``lethe-fluid-block``
-      - This application solves the Navier-Stokes equations in a monolithic way using block preconditioner.
-    * - ``lethe-fluid-matrix-free``
-      - This application solves the Navier-Stokes equations in a monolithic way and using a matrix-free approach. 
-    * - ``lethe-fluid-nitsche``
+    * - * Single-phase flows
+        * Fluid-fluid flows 
+        * Heat transfer
+        * Tracer
+      - ``lethe-fluid``
+      - This application solves the Navier-Stokes equations in a monolithic way. **This is the most robust fluid dynamics solver in Lethe  and it is compatible with all multiphysics features**.
+    * - * Single-phase flows
+        * Heat transfer
+        * Tracer
+      - ``lethe-fluid-matrix-free``
+      - This application solves the Navier-Stokes equations in a monolithic way and using a matrix-free approach.  **This is the fastest fluid dynamics solver in Lethe. This solver should be used for high-order simulations. This solver does not fully support all multiphysics features**.
+    * - * Single-phase flows
+        * Fluid-fluid flows 
+        * Heat transfer
+        * Tracer
+        * Immersed boundaries
+      - ``lethe-fluid-nitsche``
       - This application solves the Navier-Stokes equations by imposing immersed boundary conditions without discretizing the boundaries using the Nitsche method. 
-    * - ``lethe-fluid-particles``
+    * - * Particle-laden flows
+      - ``lethe-fluid-particles``
       - This application allows to run unresolved Computational Fluid Dynamics-Discrete Element Method simulations.
-    * - ``lethe-fluid-sharp``
-      - This application solves the Navier-Stokes equations by imposing immersed boundary conditions without discretizing the boundaries using the sharp-edge method. 
-    * - ``lethe-fluid-vans``
+    * - * Particle-laden flows
+        * Immersed boundaries
+      - ``lethe-fluid-sharp``
+      - This application solves the Navier-Stokes equations by imposing immersed boundary conditions without discretizing the boundaries using the sharp-edge method. It is used for resolved Computational Fluid Dynamics-Discrete Element Method simulations.
+    * - * Single phase volume-averaged flows
+      - ``lethe-fluid-vans``
       - This application solves the Volume-Averaged Navier-Stokes equations in a monolithic way.
```

Comment:
For me, it is unclear the difference between `lethe-fluid-particles` and `lethe-fluid-vans`.